app/views/home_page/_plates.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="products-wrapper">
+  <div class="plates">
+    <ul id="products">
```

Comment:
Again, minor stuff but here the naming is not really consistent as you have "plates" then "products" then "plate"? 

For me "plates-container" then "plates-list" then "plate" would sound better